docs/operations/generate_pipeline_config.rst

```diff
@@ -41,16 +41,28 @@ Assuming set up for local development fill out ``bakery/env/local.json``.
 
 .. code-block::
 
-    "S3_PDF_BUCKET": "my-bucket"
+    "S3_PDF_BUCKET": "<your-bucket>",
+    "S3_ACCESS_KEY_ID": "<your-s3-access-key>",
+    "S3_SECRET_ACCESS_KEY": "<your-s3-access-secret>",
+
+**your-bucket**: S3 Bucket where PDF will be stored.
 
 **For Distribution Pipeline:**
 
 .. code-block::
 
-    "S3_DIST_BUCKET": "ce-contents-cops-distribution-373045849756",
-    "VERSIONED_FILE": "distribution_feed.json"
+    "S3_ACCESS_KEY_ID": "<your-s3-access-key>",
+    "S3_SECRET_ACCESS_KEY": "<your-s3-access-secret>",
+    "S3_DIST_BUCKET": "<your-bucket>",
+    "VERSIONED_FILE": "<json-version-file>"
```

Comment:
A couple of points of feedback on description of configuration options here in case you think it warrants massaging this a bit:

* I added documentation on the distribution pipeline specific config in [`docs/distribution/steps`](https://github.com/openstax/output-producer-service/blob/master/docs/distribution/steps.rst#2-define-pipeline-environment-variables) which conflicts a bit with this since it references the latest version in git with different options for the distribution pipeline (e.g. `VERSIONED_FILE` is now gone).

* In general, would it make sense to have this documentation page focused on generating the configurations and let the pipeline specific (e.g. `docs/distribution/steps` and `docs/pdf/steps`) cover the configuration options?

Reply:
first point:  ah yes, you're right I saw this.
second point: yes,  docs have been updated. 

thank you!